src/context.rs

```diff
@@ -237,6 +238,36 @@ impl Context {
         self.execute_with_sessions((None, None, None), f)
     }
 
+    /// A special case of execute_with_sessions. Executes the closure with a newly generated empty session
+    pub fn execute_with_nullauth_session<F, T>(&mut self, f: F) -> Result<T>
+    where
+        // We only need to call f once, so it can be FnOnce
+        F: FnOnce(&mut Context) -> Result<T>,
+    {
+        let session = match self.start_auth_session(
+            None,
+            None,
+            None,
+            SessionType::Hmac,
+            Cipher::aes_256_cfb(),
```

Comment:
Just FYI - in the future this might need to be made configurable (maybe storing a "default cipher" on `Context`), because not all TPMs support 256 bit AES. We've had problems with that for Parsec

Reply:
I might just set it to 128 bit AES then for now.
This is a nullauth session, so not used for any actual authentication anyway.